### ISSUE TITLE
Deprecation. cant install the kohya.

### ISSUE THREAD

**Issue by TheWorl4** [bmaltais/kohya_ss#2891]
So, in short I'm tying to install kohya, but thing is that i can't... it says methos is deprecated, i guess setup.py and it is asking me to add pyproject.toml. I dont know if anyone had this problem before. i can't find soulution as well becouse i don't know how to add any toml or whataver it is... T_T 

I would greatly appreciate the help. i really want to use this amazing tool... infact i would love to exept i can not install it.

This is what it says when i open setup and try to install kohya_ss GUI: 

"DEPRECATION: Legacy editable install of library==0.0.0 from file:///C:/AI/Kohya/kohya_ss/sd-scripts (from -r requirements.txt (line 35)) (setup.py develop) is deprecated. pip 25.0 will enforce this behaviour change. A possible replacement is to add a pyproject.toml or enable --use-pep517, and use setuptools >= 64. " just as i mentioned.

"Configuration already exists at C:\Users\User/.cache\huggingface\accelerate\default_config.yaml, will not override. Run `accelerate config` manually or pass a different `save_location`." also this one as well, i dont know if this is problem or not.

In short i really need some help... Sorry for being bothersome.

**Comment by TheWorl4** [comment]
ython 3.10.11
GIT 2.46.0
CUBA 11.8

**Comment by bmaltais** [comment]
The deprecation warning is notmal and is coming from the sd-scripts setup files... I can't fix that one ;-) kohya will need to address it in his repo.

For accelerate, have you tried to run the setup option 5 to manually configure accelerate? This is essentially what accelerate is looking for. Give that a try.

**Comment by TheWorl4** [comment]
Fixed ! 

THANK YOU !
